platform/spire/src/setup.py

```diff
@@ -226,7 +226,13 @@ def modify_dns_bootstrap(ops: Operations, is_install: bool) -> None:
 
 def dns_bootstrap_lines() -> str:
     config = configuration.get_config()
-    return "".join("%s\t%s # AUTO-HOMEWORLD-BOOTSTRAP\n" % (ip, hostname) for hostname, ip in config.dns_bootstrap.items())
+    dns_hosts = list(config.dns_bootstrap.items())
```

Comment:
Currently, it looks like we'll just generate a bad `/etc/hosts` file if the same hostname is specified in setup.yaml and generated here. Can we explicitly fail in that situation, or maybe explicitly override one or the other, if you'd prefer that behavior?